utilities/dut-data-exporter/dutDataExporter.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+# Copyright (C) 2019 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import requests
+from urllib.parse import urlparse
+import pprint
+
+
+def url_port_number(url):
+    result = urlparse(url)
+    try:
+        if result.port:
+            return result.port
+        else:
+            return 80
+    except ValueError:
+        print("IPv6 address could not be parsed: {}".format(url))
+        return None
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--ip", required=True, help="IP address or Hostname of DuT")
+    parser.add_argument("--port", type=int, required=True, help="Port number of IS-04 API of DuT")
+    args = parser.parse_args()
+
+    is04Port = args.port
+    is05Port = None
+    is08Port = None
+    node_id = None
+    senders = []
+    receivers = []
+    mac_addresses = []
+
+    base_url = "http://{}:{}/x-nmos/node/v1.2/".format(args.ip, args.port)
```

Comment:
This was fine for JT-NM Tested August 2019, but should version now be made configurable on the command line, like `--version v1.3`?